nexus_integration_tests/config/zenoh/nexus_network_config.yaml

```diff
@@ -2,32 +2,68 @@
 # Use 'client' when there is an available Zenoh router, otherwise
 # use 'peer' for a distributed system.
 mode: peer
-# When true, discovery info is forwarded to the remote plugins/bridges
-forward_discovery: false
 # When true, activates a REST API used to administer Zenoh Bridge configurations
 enable_rest_api: true
-# Additional endpoints to allow, these could be endpoints not defined in REDF but are necessary for lifecycle transitions
-add_allowed_endpoints: ["/list_workcells", "/.*/workcell_state", "/.*/is_task_doable", "/.*/request", "/.*/queue_task", "/.*/remove_pending_task", "/register_workcell", "/register_transporter", ".*/available", ".*/transport", "/estop", "/.*/pause", "/.*/get_state", "/.*/change_state", "/.*/signal"]
 
 system_orchestrators:
-  - ros_namespace: system_orchestrator # ROS Namespace of the endpoints
+  - namespace: system_orchestrator # ROS Namespace of the endpoints
     # ROS Domain ID
-    domain_id: 14
+    domain_id: 0
     # Listening TCP Address of Zenoh bridge
     tcp_listen: ["0.0.0.0:7447"]
     # HTTP Port for the REST API
     rest_api_http_port: 8000
+    allow:
+      publishers: []
+      # TODO(luca) check if we need estop
+      subscribers: ["/estop", "/.*/workcell_state"]
+      service_servers: ["/list_workcells", "/register_workcell", "/register_transporter"]
+      # TODO(luca) check if transporter needs available endpoint
+      service_clients: ["/.*/queue_task", "/.*/remove_pending_task", "/.*/pause", "/.*/signal", "/.*/get_state", "/.*/change_state", "/.*/is_task_doable"]
+      action_servers: []
+      # TODO(luca) check if we really need transporter client
```

Comment:
Any idea if these TODOs are still valid?

Reply:
The demo / integration test works well but the configuration of these endpoints depends on where nodes are ran / how complete we want this config file.
The main crux is that in this specific implementation the system orchestrator and the transporter node run in the same domain ID so _technically_ it is not necessary to allow all the transporter interfaces through Zenoh.
On the other hand, if we want to have this file be kind of a template that would also allow external transporters to connect to the system orchestrator we would need to add their interfaces here.

For the `estop` I'm just not 100% sure how it is supposed to work, since it seems it is a global topic do we just halt the whole system as soon as _one_ workcell decides to estop? That sounds a bit dangerous so I left a note here.

I'm erring on the side of "make this a template" at least until a proper redf integration for generation of config files is not done and permissively allow all the interfaces we might need but what do you think?

Reply:
> For the estop I'm just not 100% sure how it is supposed to work, since it seems it is a global topic do we just halt the whole system as soon as one workcell decides to estop? That sounds a bit dangerous so I left a note here.

Yeah we halt processes across all workcells if the estop is triggered. This was a requirement.